.github/workflows/CI-main.yml

```diff
@@ -49,7 +49,7 @@ jobs:
 
       - name: Calculate version
         if: ${{ inputs.build-number == '' || inputs.build-version == '' }}
-        uses: bitwarden/ios/.github/actions/dispatch-and-download@main
+        uses: ./.github/actions/dispatch-and-download/action.yml
```

Comment:
This will run a local version of the file, which requires checking out the repo first. We're not doing that here so, [it fails](https://github.com/bitwarden/ios/actions/runs/12033334394).

Reply:
I guess because its not in the `workflow` directory, it shouldn't need to be checked out before references. We have a lot of those scenerios in `devops` [repo](https://github.com/bitwarden/devops/blob/main/.github/workflows/deploy-k8s.yml). I will update.

Reply:
We heavily use local path in all our workflows so the CLI made room for that, using local path is not causing any other problem